test/TestSites/StartupNtlmAuthentication.cs

```diff
@@ -53,18 +59,18 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
                     }
                     else
                     {
-                        return context.Authentication.ChallengeAsync();
+                        return context.ChallengeAsync();
                     }
                 }
 
                 if (context.Request.Path.Equals("/Forbidden"))
                 {
-                    return context.Authentication.ForbidAsync(AuthenticationManager.AutomaticScheme);
+                    return context.ForbidAsync();
                 }
 
                 if (context.Request.Path.Equals("/AutoForbid"))
                 {
-                    return context.Authentication.ChallengeAsync();
+                    return context.ChallengeAsync();
                 }
 
                 if (context.Request.Path.Equals("/RestrictedNegotiate"))
```

Comment:
You can remove RestrictedNegotiate, it's not used in the tests.
https://github.com/aspnet/IISIntegration/blob/e2537e1d96f4a74c8e16d067b21eb1fc6455643e/test/IISIntegration.FunctionalTests/NtlmAuthentationTest.cs#L97-L102